whipper/command/mblookup.py

```diff
@@ -0,0 +1,44 @@
+from whipper.command.basecommand import BaseCommand
+from whipper.common.mbngs import musicbrainz
+
+
+class MBLookup(BaseCommand):
+    summary = "lookup MusicBrainz entry"
+    description = """Look up a MusicBrainz disc id and output information.
+
+You can get the MusicBrainz disc id with whipper cd info.
+
+Example disc id: KnpGsLhvH.lPrNc1PBL21lb9Bg4-"""
+
+    def add_arguments(self):
+        self.parser.add_argument(
+            'mbdiscid', action='store', help="MB disc id to look up"
+        )
+
+    def do(self):
+        try:
+            discId = unicode(self.options.mbdiscid)
+        except IndexError:
+            print 'Please specify a MusicBrainz disc id.'
+            return 3
+
+        metadatas = musicbrainz(discId)
+
+        print '%d releases' % len(metadatas)
+        for i, md in enumerate(metadatas):
+            print '- Release %d:' % (i + 1, )
+            print '    Artist: %s' % md.artist.encode('utf-8')
+            print '    Title:  %s' % md.title.encode('utf-8')
+            print '    Type:   %s' % md.releaseType.encode('utf-8')  # noqa: E501
+            print '    URL: %s' % md.url
```

Comment:
The other outputs has the `%s` indented to align with each other, so this one will look out of place when printed. (This seems to be a carry-over from the old code though, so should probably be fixed in its own commit (if not PR).)

Reply:
good catch-- should be fixed in this PR.